### ISSUE TITLE
Emitting from external process back to main server only

### ISSUE THREAD

**Issue by tsoernes** [miguelgrinberg/Flask-SocketIO#1503]
Is it possible to emit an external process, running on Redis, back to the main server only? Does the server itself have a sid?

**Comment by miguelgrinberg** [comment]
No, this package helps you communicate clients and servers. If you have two server-side process that need to communicate you have a lots of options, like for example sending an HTTP request.
